src/WebJobs.Extensions.DurableTask/EntityScheduler/Proxy/DurableEntityProxyExtensions.cs

```diff
@@ -15,155 +15,7 @@ public static class DurableEntityProxyExtensions
-        /// <typeparam name="TEntityInterface">Entity interface.</typeparam>
-        public static void SignalEntity<TEntityInterface>(this IDurableEntityContext context, string entityKey, Action<TEntityInterface> operation)
-        {
-            SignalEntity<TEntityInterface>(context, new EntityId(ResolveEntityName<TEntityInterface>(), entityKey), operation);
-        }
-
-        /// <summary>
-        /// Signals an entity to perform an operation, at a specified time.
-        /// </summary>
-        /// <param name="context">entity context.</param>
-        /// <param name="entityKey">The target entity key.</param>
-        /// <param name="scheduledTimeUtc">The time at which to start the operation.</param>
-        /// <param name="operation">A delegate that performs the desired operation on the entity.</param>
-        /// <typeparam name="TEntityInterface">Entity interface.</typeparam>
-        public static void SignalEntity<TEntityInterface>(this IDurableEntityContext context, string entityKey, DateTime scheduledTimeUtc, Action<TEntityInterface> operation)
-        {
-            SignalEntity<TEntityInterface>(context, new EntityId(ResolveEntityName<TEntityInterface>(), entityKey), scheduledTimeUtc, operation);
-        }
-
-        /// <summary>
-        /// Signals an entity to perform an operation.
-        /// </summary>
-        /// <param name="context">entity context.</param>
-        /// <param name="entityId">The target entity.</param>
-        /// <param name="operation">A delegate that performs the desired operation on the entity.</param>
-        /// <typeparam name="TEntityInterface">Entity interface.</typeparam>
-        public static void SignalEntity<TEntityInterface>(this IDurableEntityContext context, EntityId entityId, Action<TEntityInterface> operation)
-        {
-            operation(EntityProxyFactory.Create<TEntityInterface>(new EntityContextProxy(context), entityId));
-        }
-
-        /// <summary>
-        /// Signals an entity to perform an operation, at a specified time.
-        /// </summary>
-        /// <param name="context">entity context.</param>
-        /// <param name="entityId">The target entity.</param>
-        /// <param name="scheduledTimeUtc">The time at which to start the operation.</param>
-        /// <param name="operation">A delegate that performs the desired operation on the entity.</param>
-        /// <typeparam name="TEntityInterface">Entity interface.</typeparam>
-        public static void SignalEntity<TEntityInterface>(this IDurableEntityContext context, EntityId entityId, DateTime scheduledTimeUtc, Action<TEntityInterface> operation)
-        {
-            operation(EntityProxyFactory.Create<TEntityInterface>(new EntityContextProxy(context, scheduledTimeUtc), entityId));
-        }
-
-        private static string ResolveEntityName<TEntityInterface>()
+        public static string ResolveEntityName<TEntityInterface>()
```

Comment:
This class should probably be renamed now.

Reply:
Would it make sense to combine it with our other extension method class so that we have just one?

Reply:
I renamed the class, as it didn't make sense to put it in the extension method class since this isn't an extension method.